src/common/helpers/time.ts

```diff
@@ -0,0 +1,25 @@
+interface Time {
```

Comment:
```suggestion
interface Duration {
```
Even though this is not exported, duration is more appropriate name for this time type.